docs/script-catalog/fido2_interception/fido2-interception/Fido2Interception_Script.py

```diff
@@ -0,0 +1,66 @@
+# Janssen Project software is available under the Apache 2.0 License (2004). See http://www.apache.org/licenses/ for full text.
+# Copyright (c) 2023, Janssen Project
+#
+# Author: Jorge Munoz
+#
+
+from io.jans.service.cdi.util import CdiUtil
+from io.jans.model.custom.script.type.auth import Fido2InterceptionType
+from io.jans.fido2.service.operation import AttestationService
+from io.jans.fido2.service.operation import AssertionService
+from io.jans.util import StringHelper
+from io.jans.as.model.error import ErrorResponseFactory
+from jakarta.ws.rs.core import Response
+from io.jans.as.model.authorize import AuthorizeErrorResponseType
+from org.json import JSONObject
+from com.fasterxml.jackson.databind import JsonNode
+
+import java
+
+class Fido2Interception(Fido2InterceptionType):
```

Comment:
Did you try this script ? According to `CustomScriptType` jython will fail to load it.